homework/length-sort/sort.cpp

```diff
@@ -0,0 +1,21 @@
+#include "sort.hpp"
+
+bool compareSize(const std::string &a, const std::string &b)
+{
+    if (a.size() == b.size())
+    {
+        return std::lexicographical_compare(a.begin(), a.end(), b.begin(), b.end());
```

Comment:
you can compare string using operator==.

str1 == str2 will be automatically compared lexicographical :)